docs/whats-new/dotnet-2020-04.md

```diff
@@ -0,0 +1,124 @@
+---
+title: ".NET docs: What's new for April 2020"
+description: "What's new in .NET docs for April 2020."
+ms.date: 04/27/2020
+---
+
+# .NET docs: What's new for April 2020
+
+Welcome to what's new in .NET docs for April 2020. This article lists some of the major changes to docs during this period.
+
+## Architecture guides
+
+### New articles
+
+- [gRPC](../architecture/cloud-native/grpc.md) - Cloud Native - Chapter 5 - Updates to gRPC
+
+### Updated articles
+
+- [App configuration](../architecture/blazor-for-web-forms-developers/config.md) - Blazor ebook config
+
+## .NET Core
+
+### New articles
+
+- [Trim self-contained deployments and executables](../core/deploying/trim-self-contained.md) - Create trimming doc
+- [Ubuntu 20.04 Package Manager - Install .NET Core](../core/install/linux-package-manager-ubuntu-2004.md) - Install for Ubuntu 20.04
```

Comment:
Given #18092, should it be here?
/cc @Thraka

Reply:
Yeah remove that. Also, with creating the PR for review prior to the sprint end but accepted today (#18080)[https://github.com/dotnet/docs/pull/18080] Does that appear next sprint whats new or this one?